### ISSUE TITLE
[Bug]: TMDB Not Working (Failed to Fetch Data))

### ISSUE THREAD

**Issue by heyitsleo** [avalynndev/enjoytown#171]
### What went wrong?

Loading the homepage, returns nothing under the carasoul. "Host Not Permitted"

### How'd you stumble upon it?

1. I was on the homepage
2. I was updating my NavBar
3. I re-opened the project today and checked if it was happening on the website (deployed by you)

### What you hoped for vs. What actually happened

Showing the data required for the pages,
Happened: Nothing

### Anything else we should know?

Anime, and Drama pages are also broken.

### Which browsers were a bummer?

Firefox, Chrome, Safari, Microsoft Edge

### Got logs?

```shell
movies.ts:93 
 GET http://localhost:3000/api/tmdb/movie/popular?language=en-US 403 (Forbidden)

tv-series.ts:129 
 GET http://localhost:3000/api/tmdb/tv/popular?language=en-US 403 (Forbidden)
```

### Code of Conduct Agreement

- [X] I agree to follow this project's Code of Conduct.

**Comment by avalynndev** [comment]
API is down, don't have money to bring it back up.. if I people are interested in donating to this project I would be glad to bring it back

**Comment by avalynndev** [comment]
@ctrlsam.. noticed it was due to the update you have given.. could you work on this issue?

**Comment by heyitsleo** [comment]
update: i managed to fix the home page part, somehow me fixing that brought back the details page. 

to be fixed: actual pages: /moive, /tv, and for tv: video player stuff.

**Comment by avalynndev** [comment]
guess ITZ cuz of the new type of fetching.. try to fix u want. I wont be working on this project as I have no money to keep it up. so let it die ig..

**Comment by ctrlsam** [comment]
@heyitsleo Is there more context here around the 403? Perhaps a text response from the API describing why it's returning that status code? @avalynndev what do you mean by "no money to keep it up" - have you hit an API limit? As for your question, I don't have time to work on this now, but consider adding caching on the requests so that the API key stays within it's usage limit is not over used.

**Comment by ctrlsam** [comment]
Looking at the request, I'm assuming TMDB have banned your domain name. This wasn't a consideration when porting TMDB api calls to the serverside, but arguably still better than exposing your api key. If you can ask them about the error we may get some insights into how to proceed.

**Comment by heyitsleo** [comment]
I'm not sure tmdb banned my domain as I can still access it the old way. It's also broken on the demo website and other forks of enjoytown.

**Comment by heyitsleo** [comment]
> guess ITZ cuz of the new type of fetching.. try to fix u want. I wont be working on this project as I have no money to keep it up. so let it die ig..

I'm willing semi-keep it going, I do provide daily changes to my own version of the code and the api is still running, so

**Comment by heyitsleo** [comment]
I could invite you to my repository so you can see the changes if you'd want.

**Comment by FzCDN** [comment]
> API is down, don't have money to bring it back up.. if I people are interested in donating to this project I would be glad to bring it back

if you can use my server if you can help for sub indonesia

**Comment by ctrlsam** [comment]
I think the "old way" works because it's fetched from your browser and therefore not via the backend server

**Comment by heyitsleo** [comment]
> I think the "old way" works because it's fetched from your browser and therefore not via the backend server

Probably the case. If there is a work around; LMK. Otherwise everything runs perfectly fine for me.

**Comment by Svznify** [comment]
Issue has been fixed: #180
